src/hotspot/share/runtime/objectMonitor.hpp

```diff
@@ -306,15 +306,17 @@ class ObjectMonitor : public CHeapObj<mtInternal> {
   class ExitOnSuspend {
    protected:
     ObjectMonitor* _om;
-    bool _om_op_done;
+    bool _exited;
```

Comment:
For consistency with raw monitor code this would _om_exited

Reply:
Fixed